book/src/api-lighthouse.md

```diff
@@ -192,9 +192,7 @@ health of the Eth1 node that the beacon node is connected to.
 number and timestamp of the latest block we have in our block cache.
 	- For correct Eth1 voting this timestamp should be later than the
 `voting_period_start_timestamp`.
-- `voting_period_start_timestamp`: the start of the period where block
-	producers must include votes for blocks in the Eth1 chain. Provided for
-	reference.
+- `voting_target_timestamp`: The latest timestamp allowed for an eth1 block in this voting period.
```

Comment:
"latest" seems ambiguous here. We mean "oldest" don't we? Not "most recent"?

Reply:
We do mean most recent.

In an eth1 voting period we are permitted to vote on a range of timestamps. This is the highest timestamp in that range.